hexrd/ui/indexing/run.py

```diff
@@ -85,6 +85,12 @@ def ome_maps_selected(self):
         else:
             # Create a full indexing config
             config = create_indexing_config()
+            # Save the hkls strings for future comparison
```

Comment:
If the user chooses to load a file, maybe we can set `omaps['_active_hkl_strings'] = None`, where `None` indicates that a file was loaded?